qemu/tests/single_driver_install.py

```diff
@@ -127,7 +127,11 @@ def run(test, params, env):
                                           pnp_cmd) % inf_name
             status, output = session.cmd_status_output(uninst_store_cmd,
                                                        inst_timeout)
-            if status:
+            if params.get_boolean('need_reboot'):
+                # for viostor and vioscsi, they need system reboot
+                # acceptable status: OK(0), REBOOT(3010)
+                session = vm.reboot(session)
+            if status not in (0, 3010):
```

Comment:
@menli820 Could you please move this before L130?

Reply:
@xiagao update, thanks

 (1/1) Host_RHEL.m9.u5.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.win_virtio_driver_update_test.with_viostor.during_bg_test.uninstall_install.q35: STARTED
 (1/1) Host_RHEL.m9.u5.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.win_virtio_driver_update_test.with_viostor.during_bg_test.uninstall_install.q35: PASS (440.53 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0